### ISSUE TITLE
Provide a mechanism to override the navbar-brand text in ermheader.html

### ISSUE THREAD

**Issue by mikedarcy** [informatics-isi-edu/chaise#269]
Similar to the **headTitle** parameter in chaise-config.js, it would be great to have a parameter to control the text that appears in the bootstrap navbar.  It is currently hardcoded to Chaise in ermheader.html.  

There is already code to override the navbar link, so adding some to override the text shouldn't be too tough and will be very useful for further customization of deployments.

**Comment by svoinea** [comment]
I think this is currently done through the 'customCSS' parameter. For example, in RBK, we have:

    customCSS: '/assets/css/chaise.css',

and in the /var/www/html/assets/css directory, the content of the chaise.css file is:

/* Customization styles for Chaise/Image Annotation ap */

.navbar-brand span {
  display: none;
}

.navbar-brand:after {
  content: 'RBK';
}

.navbar-inverse {
  background-color: #002d73;
}

**Comment by mikedarcy** [comment]
OK, I will try this out and if it works I'll close the issue.  Having this example in the configuration section of the docs would be helpful.

**Comment by crisaless** [comment]
This does work - but @hongsudt  and I were recently discussing that it would be more straightforward and consistent if you could use a config variable like "headTitle" instead of the css workaround. So it's not urgent, but something we should do at some point.

**Comment by robes** [comment]
Right. I would agree with that. Changing the CSS is still an overwrite rather than a configuration change.

**Comment by robes** [comment]
Sorry, my last comment wasn't correct. I didn't notice that this was a `customCSS` field, so its not overwriting any standard file.

**Comment by svoinea** [comment]
Added the navbarBrandText parameter in chaise_config.js. It replaces the hard coded "Chaise" that appears in the bootstrap navbar.

**Comment by hongsudt** [comment]
@mikedarcy Can you review this issue, and if it is sufficient, please close it.

**Comment by mikedarcy** [comment]
This doesn't seem to work on Chrome (does work on FF).  Also, it shows the default hard-coded value of "Chaise" before the swapping code modifies the DOM.  This has the effect of being just a flash on systems where the page load is quick (because the facet list is short), but when the facet list is long the default string is clearly visible while the page renders.

**Comment by mikedarcy** [comment]
We also really need a way to add an image to the navbarBrand, e.g.:

`              <a class="navbar-brand" href="#">
                <img src="../images/bdds_logo_final.png">
                <span>BDDS Data Explorer</span>
              </a>
`
If both this and the text swap can be done via chaise-config.js, great.  If it has to be done via CSS that is acceptable.  I can file a separate issue for the image swap if need be.

**Comment by mikedarcy** [comment]
Serban fixed this so it works on Chrome and most likely other browsers too.

I added a way to override the navbar image using the same mechanism.  Set "navbarBrandImage" in chaise-config.js to a webroot relative path (similar to "customCSS") and it works fine.  You will likely need to use "customCSS" as well, in order to size the image properly if it needs to be scaled.  

Here's what I did,YMMV:

`.navbar-brand>img {
  height: 100%;
  width: auto;
}
`
